### ISSUE TITLE
<Link><FormattedMessage /></Link> doesn't work. 

### ISSUE THREAD

**Issue by elivoa** [umijs/umi#49]
1. 
```javascript
<Link to="/test-page" className={styles.test}>
  <FormattedMessage id="lang.message" />
</Link>
```

will generate the following HTML, can't click.
```html
<span>some message</span>
```

2.
```javascript
<Link to="/test-page" className={styles.test}>
  <span>some message</span>
</Link>
```
will generate the same HTML as in (1.), it can be clicked, but with no `pointer` cursor.

3.
```javascript
<Link to="/test-page" className={styles.test}>
  <a><FormattedMessage id="lang.message" /></a>
</Link>
```

this works fine.

I think it's a bug.

Another thing, Link only accept one child, the following code will throw an exception.
```javascript
<Link to='xxx'>
  <Icon class=''/> Message
</Link>
```

**Comment by sorrycc** [comment]
Thanks for report, we'll switch to react-router@4 for router later.

**Comment by elivoa** [comment]
Waiting for that. I can continue migration my project when it's ready.
Thank you for your awesome framework.

**Comment by sorrycc** [comment]
Fixed via umi@1.0.0-beta.35 .
